tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core

```

Comment:
Ah, yes, didn't pay attention to that. Actually, `CallOrderingContext` was initially in `FakeItEasy.Core`, I moved it to `FakeItEasy` when I removed the interface, then forgot to move it back when I moved the interface again. Should I move the class, or the test? I'm never sure what should be in the root namespace or in the `Core` namespace...